test/connectors/exchange/ExportsDeclarationExchangeSpec.scala

```diff
@@ -23,12 +23,12 @@ import forms.declaration.additionaldeclarationtype.AdditionalDeclarationType.Add
 import forms.{Ducr, Lrn}
 import models.declaration._
 import models.{DeclarationStatus, DeclarationType, ExportsDeclaration}
-import org.scalatest.{Matchers, WordSpec}
+import base.UnitSpec
 import org.scalatestplus.mockito.MockitoSugar
 import play.api.libs.json.{JsSuccess, Json}
 import services.cache.ExportsDeclarationBuilder
 
-class ExportsDeclarationExchangeSpec extends WordSpec with Matchers with ExportsDeclarationBuilder with MockitoSugar {
+class ExportsDeclarationExchangeSpec extends UnitSpec with ExportsDeclarationBuilder with MockitoSugar {
```

Comment:
Shouldn't this also be UnitWithMocksSpec

Reply:
Good spot :+1: